macros/test-helpers/check_seed_macro.sql

```diff
@@ -54,8 +54,9 @@
     matching_count_test as (
         select
             'matched records count' as test_description,
-            count(model_{{seed_matching_columns[0]}}) as `result_model`,
-            1 as `expected_seed`,
+            cast(count(model_{{seed_matching_columns[0]}}) as varchar) as result_model,
+            cast(1 as varchar) as expected_seed,
```

Comment:
Why  the casts here?

Reply:
Ah, makes sense. Could we add a comment on this? E.g., `cast all columns to varchar to unify column types`